src/pages/authentication-pages/signin-page/SignInPage.jsx

```diff
@@ -0,0 +1,97 @@
+import { Link } from "react-router-dom";
+import { useAuthenticationContext } from "context";
+import { submitSignInDetails } from "utility";
+import { useNavigate } from "react-router-dom";
+import "./signin-page.css";
+
+const SignInPage = () => {
+  // getting encodedToken from localStorage
+  const encodedToken = localStorage.getItem("token");
+
+  // Authentication Context
+  const {
+    authState: { email, password },
+    authDispatch,
+  } = useAuthenticationContext();
+
+  // from react-router-dom
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <main className="sign-in">
+        <div className="input-container sign-in-container">
+          <h2 className="text-center">SignIn</h2>
+          <form className="input-flex">
+            <label htmlFor="email">Email Id</label>
+            <input
+              className="input"
+              type="email"
+              id="email"
+              name="email"
+              required
+              onChange={(e) =>
+                authDispatch({ type: "EMAIL", payload: e.target.value })
+              }
```

Comment:
I am not sure if this is fine or bad practice, you are using onChange without a value attribute on the input field. I will read more about it.